src/components/toggle/toggle.html

```diff
@@ -0,0 +1,13 @@
+<label [attr.for]="inputId" class="md-toggle-label">
```

Comment:
I think it makes sense as there is also the toggle slider, I'm fine with changing it but will wait for a go-ahead.